hal/lib/athena/handles/HandlesInternal.cpp

```diff
@@ -13,9 +13,10 @@
 namespace hal {
 HalHandle createHandle(int16_t index, HalHandleEnum handleType) {
   if (index < 0) return HAL_HANDLE_NEGATIVE_INDEX;
-  if (handleType <= 0 || handleType > 127) return HAL_HANDLE_INVALID_TYPE;
+  uint8_t hType = (uint8_t)handleType;
```

Comment:
We prefer C++ style casts (e.g. `static_cast<uint8_t>(handleType)`) over C-style.  Same comment applies to the uint16_t casts added other places.